src/types/db/training.day.types.ts

```diff
@@ -3,8 +3,8 @@ import { Document, Model, ObjectId } from 'mongoose';
 export type TTrainingDay = {
     _id: ObjectId,
     name: string,
-    excercises: {
-        excerciseId: string,
+    exercises: {
+        exerciseId: string,
```

Comment:
```suggestion
        exerciseId: Schema.Types.ObjectId,
```

Sieht für mich nach einer Referenz auf ein anderes Model aus. Außerdem klingt mir exercise**s** eher nach einem Array